modules/services/analytics/src/main/java/au/com/shiftyjelly/pocketcasts/analytics/AnalyticsEvent.kt

```diff
@@ -655,5 +655,14 @@ enum class AnalyticsEvent(val key: String) {
     TRANSCRIPT_SEARCH_PREVIOUS_RESULT("transcript_search_previous_result"),
 
     /* Referrals */
-    REFERRAL_LINK_SHARED("referral_link_shared"),
+    REFERRAL_TOOLTIP_SHOWN("referral_tooltip_shown"),
+    REFERRAL_TOOLTIP_TAPPED("referral_tooltip_tapped"),
+    REFERRAL_SHARE_SCREEN_SHOWN("referral_share_screen_shown"),
+    REFERRAL_PASS_SHARED("referral_pass_shared"),
+    REFERRAL_CLAIM_SCREEN_SHOWN("referral_claim_screen_shown"),
```

Comment:
We can close the claim screen by using the drag down gesture. Do you think we need to track this too?

Reply:
Good catch!
According to the description, it should be tracked when `User taps the link on the claim screen "Not Now" or uses the back button`

I fixed it in de00a19.